packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -1,13 +1,24 @@
 import { useStore } from 'shared/libs/stores';
 
 import { isEqual } from 'lodash';
+import isPast from 'date-fns/isPast';
+
+import { AccessRequest } from 'e-teleport/services/workflow';
```

Comment:
In general, any part of the UI that uses an `AccessRequest` is pulling from the e-teleport type, any response to us is our gRPC type. The difference being our types append `List` to the end of any repeated field sent back in gRPC and this was a bandaid I never ripped off. I think its ugly, however was a simpler fix than going down the grpc rabbithole and why that `List` was being appended. I couldn't find it but maybe you have an idea of where its at? Also, my thought process was "if I remove list for this request, it would probably destroy the rest of the app that relied on that append".

also, fwiw, created/expires are used in `makeAccessRequest` in e-teleport to convert json to Date objects
https://github.com/gravitational/webapps.e/blob/9498eaa8c9893853d72c09e604940600386853d5/teleport/src/services/workflow/makeAccessRequest.ts#L20-L22

Conversion happens here with an input type I named `TshdAccessRequest` 
https://github.com/gravitational/webapps.e/blob/36b095b107e85f20782f72cd1723e511cbfe534c/teleterm/src/ui/DocumentAccessRequests/useAccessRequests.tsx#L124-L133

Reply:
> The difference being our types append `List` to the end of any repeated field sent back in gRPC and this was a bandaid I never ripped off. I think its ugly, however was a simpler fix than going down the grpc rabbithole and why that `List` was being appended. I couldn't find it but maybe you have an idea of where its at?

It looks like that's how the code which gets autogenerated works. Maybe there's a config option to change that but yeah, at this point it's better to work around that.

It might make sense though to manually copy over fields rather than using a spread there, just to make finding fields by reference easier. This way someone trying to find usage of AccessRequest.id or AccessRequest.resolveReason will see where and how they're actually used.